library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -232,6 +232,12 @@ public int hashCode()
 
     protected ConcurrentLinkedQueue<Long> windowsForFreeMemory = new ConcurrentLinkedQueue<>();
 
+    private static boolean disableBloomFilterByDefault = false;
```

Comment:
My intension is giving the client code a choice to use or disable the bloom filter for all bucket. As we don't have a central place to manage the instances. It's hard to get the instance for app developer. I'll change it into public